api/core/number.go

```diff
@@ -190,74 +202,106 @@ func (n Number) CoerceToUint64(kind NumberKind) uint64 {
 
 // - set
 
+// SetNumber sets the number to the passed number. Both should be of
+// the same kind.
 func (n *Number) SetNumber(nn Number) {
 	*n.AsRawPtr() = nn.AsRaw()
 }
 
+// SetRaw sets the number to some raw value. Both number and the raw
+// number should represent the same kind.
 func (n *Number) SetRaw(r uint64) {
 	*n.AsRawPtr() = r
 }
 
+// SetInt64 assumes that the number contains an int64 and sets it to
+// the passed value.
 func (n *Number) SetInt64(i int64) {
 	*n.AsInt64Ptr() = i
 }
 
+// SetFloat64 assumes that the number contains a float64 and sets it
+// to the passed value.
 func (n *Number) SetFloat64(f float64) {
 	*n.AsFloat64Ptr() = f
 }
 
+// SetUint64 assumes that the number contains a uint64 and sets it to
+// the passed value.
 func (n *Number) SetUint64(u uint64) {
 	*n.AsUint64Ptr() = u
 }
 
 // - set atomic
 
+// SetNumberAtomic atomically sets the number to the passed
+// number. Both should be of the same kind.
```

Comment:
maybe rephrase it `SetNumberAtomic sets the number to the passed number atomically. Both should be of the same kind.`

Reply:
Thanks. Fixed.